src/resources/js/components/AccountEdit.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <v-container>
+
+  <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+  <h2>My Account</h2>
+
+  <v-form
+    ref="form"
+    v-model="valid"
+    lazy-validation
+  >
+    <v-text-field
+      v-model="updateUserInfo.fname"
+      :counter="10"
+      :rules="nameRules"
+      label="firstName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.lname"
+      :counter="10"
+      :rules="nameRules"
+      label="lastName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.email"
+      :rules="emailRules"
+      label="E-mail"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.password"
+      :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+      :type="showPassword ? 'text' : 'password'"
+      label="password"
+      value=""
+      class="input-group--focused"
+      @click:append="showPassword = !showPassword"
+    ></v-text-field>
+
+    <v-btn
+      :disabled="!valid"
+      color="success"
+      class="mr-4"
+      @click="submitChange"
+    >
+      Submit
+    </v-btn>
+
+  </v-form>
+
+  </v-container>
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+  name: 'AccountEdition',
+  props: ['parent'],
+  data() {
+    return {
+      // vuetify
+      valid: true, // if a rules is not fully satisfied this will disable submit button
+      nameRules: [
+        (v) => v === '' || v.length <= 10 || 'Name must be less than 10 characters',
+      ],
+      emailRules: [
+        (v) => v === '' || /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,})+$/.test(v) || 'E-mail must be valid',
+      ],
+      showPassword: false,
+      // object for message management
+      message: {
+        show: false,
+        content: '',
+        type: '',
+      },
+      /*
+       * For now LoggedUser is the current user logged info
+       * this will serve as temporary test and debug value
+       */
+      loggedUser: {
+        id: 3,
+        fname: 'Jane',
+        lname: 'Doe',
+        email: 'jane@paul.lo',
+      },
+      // Object for form field binding
+      updateUserInfo: {
+        fname: '',
+        lname: '',
+        email: '',
+        password: '',
+      },
+    };
+  },
+  methods: {
+    // old
+    submitChange() {
+      // API Url use mirageJS for testing,
+      const apiUrl = `/api/user/${this.loggedUser.id}`;
```

Comment:
the api url are now under the form `/api/v1/<last word>` and not `/api/<last word>`
also the to get the current logged  user we have to do GET api/v1/me, more info about all this here
https://github.com/kayoumido/HEIGVD-PRO-A-11/blob/develop/src/public/docs/index.html

Reply:
ok added in [77006](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/123/commits/eafa77006ac620603cf915decd0297cb453dfe30)